dd-java-agent/agent-bootstrap/src/main/java/datadog/trace/bootstrap/instrumentation/jdbc/JDBCConnectionUrlParser.java

```diff
@@ -925,6 +944,9 @@ private static void populateStandardProperties(
           ExceptionLogger.LOGGER.debug("Error parsing portnumber property: {}", portNumber, e);
         }
       }
+      if (props.containsKey("servicename")) {
+        builder.instance((String) props.get("servicename"));
+      }
```

Comment:
this seems unrelated ? Can we ate least mention what this does in the PR description ?

Reply:
it is not unrelated I already put a comment see https://infocenter.sybase.com/help/index.jsp?topic=/com.sybase.infocenter.dc01776.1601/doc/html/san1357754914053.html

Reply:
I still don't understand the link with the code above.
There is some specific code to parse the sybase connection string, ok. Why do we need to read the service name from props because of that ?

Reply:
The general format looks like `jdbc:sybase:Tds:host:port/database` (the customer is using this one). 
This kind of opaque URI (does not contains something like `://` is not directly parsable  by our generic parser so I need to adapt it. Concerning the service name, it is mentioned in that webpage if you click on `How to Specify a Database with a jConnect Connection String` (looks like I copied the link but when I click i'm not on that one 🤷 ). I used this part to fill up the `instance` in a similar way we do with oracle for the service name (we use the instance part of the dbinfo)

Reply:
ah ok. And we don't have access to the props in `doParse` so we do it here rather than there.
Still, I think it'd be nice to have a comment here saying that this is specific to Sybase (since it's in "common" code and not in a section that's identified as specific to sybase)
```suggestion
      if (props.containsKey("servicename")) {
        // this property is used to specify the db to use for Sybase connection strings
        builder.instance((String) props.get("servicename"));
      }
```

Reply:
yep makes sense. thanks for having suggested it